hs_core/templates/pages/my-resources.html

```diff
@@ -267,6 +268,15 @@ <h4 class="panel-title">
                                     Favorite
                                 {% endif %}
                             </td>
+                            <td>
+                                {% if res.raccess.public %}
+                                    <strong style="color:#5cb85c">&nbspPublic</strong>
+                                {% elif not res.raccess.public and res.raccess.discoverable %}
```

Comment:
"not res.raccess.public" is not needed here in the condition since it is already guaranteed by "elif"